go/vt/vtgate/planbuilder/testdata/aggr_cases.json

```diff
@@ -675,9 +675,41 @@
     }
   },
   {
-    "comment": "scatter aggregate group by aggregate function",
+    "comment": "scatter aggregate group by aggregate function - since we don't have authoratative columns for user, we can't be sure that the user isn't referring a column named b",
```

Comment:
I wanted to keep one of these around. The rest of these queries, I've instead moved to using a table that we have full column info on.